src/HPCCache/HPCCache/Az.HPCCache.psd1

```diff
@@ -0,0 +1,136 @@
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.7.0'; })
+
+# Assemblies that must be loaded prior to importing this module
+RequiredAssemblies = 'Microsoft.Azure.Management.StorageCache.dll'
+
+# Script files (.ps1) that are run in the caller's environment prior to importing this module.
+# ScriptsToProcess = @()
+
+# Type files (.ps1xml) to be loaded when importing this module
+# TypesToProcess = @()
+
+# Format files (.ps1xml) to be loaded when importing this module
+# FormatsToProcess = @()
+
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+NestedModules = @('Microsoft.Azure.PowerShell.Cmdlets.HPCCache.dll')
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = 'Get-AzHpcCacheSku', 'Get-AzHpcCacheUsageModel', 'Get-AzHpcCache',
+               'New-AzHpcCache', 'Remove-AzHpcCache', 'Set-AzHpcCache',
+               'Start-AzHpcCache', 'Stop-AzHpcCache', 'Update-AzHpcCache',
+               'Remove-AzHpcCacheStorageTarget', 'New-AzHpcCacheStorageTarget',
+               'Get-AzHpcCacheStorageTarget', 'Set-AzHpcCacheStorageTarget'
+
+# Variables to export from this module
+# VariablesToExport = @()
+
+# Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+AliasesToExport = @()
+
+# DSC resources to export from this module
+# DscResourcesToExport = @()
+
+# List of all modules packaged with this module
+# ModuleList = @()
+
+# List of all files packaged with this module
+# FileList = @()
+
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = 'Azure','ResourceManager','ARM','HPC','HPCCache','StorageCache'
+
+        # A URL to the license for this module.
+        LicenseUri = 'https://aka.ms/azps-license'
+
+        # A URL to the main website for this project.
+        ProjectUri = 'https://github.com/Azure/azure-powershell'
+
+        # A URL to an icon representing this module.
+        # IconUri = ''
+
+        # ReleaseNotes of this module
+        ReleaseNotes = '* Preview of ''Az.HPCCache'' module'
+
+        # Prerelease string of this module
+        Prerelease = 'preview'
```

Comment:
```suggestion
        # Prerelease = 'preview'
```
Preview is unnecessary if the module is to be published to the public gallery and version < 1.0.0

Reply:
Fixed in b2d9a68